pkg/proto/v0/hello.proto

```diff
@@ -1,7 +1,7 @@
 syntax = "proto3";
 
-package com.owncloud.ocis.hello.v0;
-option go_package = "proto";
+package proto;
+option go_package = ".;proto";
```

Comment:
Shouldn't `option go_package = "proto"` work as well? Using this syntax requires [some background](https://github.com/golang/protobuf/issues/139)